tools/devops/automation/templates/mac/stage.yml

```diff
@@ -32,6 +32,9 @@ stages:
   # we need to have the pkgs built and the device sets to be ran, that is decided via the labels or type of build during the build_packages stage
   condition: and(succeeded(), eq(dependencies.build_packages.outputs['build.configuration.RunDeviceTests'], 'True'))
 
+  variables:
+    MAC_TESTS_PRESENT: $[ dependencies.build_packages.outputs['build.macTestPkg.macTestsPresent'] ]
```

Comment:
This variable doesn't seem to be used anywhere?

Reply:
I'm stupid, when to fast :/

Reply:
Now it is in ' Validate tests pacakge'